pkg/cli/admin/inspect/pod.go

```diff
@@ -75,7 +75,8 @@ func (o *InspectOptions) gatherContainerInfo(destDir string, pod *corev1.Pod, co
 	}
 
 	if err := o.gatherContainerEndpoints(path.Join(destDir, "/"+container.Name), pod, &container, port); err != nil {
-		return err
+		log.Printf("        Skipping container endpoint collection for pod %q container %q: %v\n", pod.Name, container.Name, err)
+		return nil
```

Comment:
nit: looking at endpoints collection it doesn't look fatal so I'd not return here and let it continue, although that doesn't matter much when it is currently at the end of the function (can change over time)